lib/db/schema/patch-026-027.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS securityEvents (
+  uid BINARY(16) NOT NULL,
+  name VARCHAR(255) NOT NULL,
```

Comment:
I'd be happy with a reference table; mostly I'm thinking about the overall db write load and how it seems to bite us on a semi-regular basis.